<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Procurar palavra no texto</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="searchBar">
     <item>
      <widget class="QLabel" name="searchBarLabel">
       <property name="text">
        <string>Keyword</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="searchBarInput"/>
     </item>
     <item>
      <widget class="QPushButton" name="searchBarButton">
       <property name="text">
        <string>Procurar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="sourceText">
     <item>
      <widget class="QTextEdit" name="sourceTextField"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
